docs/content/python/guides/enforcement/enforcer.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Build an enforcer"
+weight: 1
+any: true
+description: >
+  Learn how to construct an Enforcer instance so that you can use its methods
+  throughout your app.
+# showContentForAnyLanguage: true
+# draft: True
+---
+
+{{% callout "Note: 0.20.0 Alpha Feature" %}}
+  This is an API provided by the alpha release of Oso 0.20.0, meaning that it is
+  not yet officially released. You may find other docs that conflict with the
+  guidance here, so proceed at your own risk! If you have any questions, don't
+  hesitate to [reach out to us on Slack](https://join-slack.osohq.com). We're
+  here to help.
+{{% /callout %}}
+
+<div class="pb-10"></div>
+
+# Construct an `Enforcer` instance
+
+To make use of the new Oso enforcement APIs, you'll need an Enforcer instance.
+The enforcer is a link between your policy and your application code, and should
+be accessible throughout your app.
+
+```python
+from oso import Enforcer, Policy
+
+def init_oso(app):
+  # Build a policy and configure it
+  policy = Policy()
+  policy.register_class(...)
+  policy.load_file(...)
+
+  oso = Enforcer(policy)
+
+  # Make the oso enforcer accessible throughout the app
+  app.oso = oso
+```
+
+An enforcer exposes a number of useful methods that your app can use. Each one
+queries your policy for `allow`, `allow_request`, or `allow_field` rules. To
+learn more about writing those rules, read about [writing a
+policy](../../getting-started/policies).
+
+- `authorize(user, action, resource)`: Ensure that a user can perform an action
```

Comment:
Would be great to link to API documentation here for each method. You can use the apiDeepLink shortcode for that.